.theme/pages/open-graph.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    {{ description }}
+    <br>
+    Open the page source code and look at the meta to see the result
+  </div>
+</template>
+
+<script setup>
+import { useHead } from "#head";
+
+const description = "This is an example of accessing meta with 'useHead' for open graph.";
```

Comment:
doküman içeriği olabilecek şeyler yazmayalım vue tarafında. description'a `sample open graph description` verebilirsin örneğin, aynı şekilde title'a da